javascript/organizing_your_javascript_code/objects_and_object_constructors.md

```diff
@@ -239,6 +237,18 @@ carl.sayName() //uh oh! this logs "HAHAHAHAHAHA" because we edited the sayName f
 
 If we had used `Object.create` in this example, then we could safely edit the `NinthGrader.prototype.sayName` function without changing the function for `EighthGrader` as well.
 
+
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+1. Read up on the concept of the prototype from the articles below. 
+   1. [This article](https://web.archive.org/web/20200513181548/https://javascriptissexy.com/javascript-prototype-in-plain-detailed-language/) is a straightforward introduction and demonstration of the concept. It also covers constructors again.. good time for a review! The important bits here, once you've covered the basics, are 'Prototype-based inheritance' and the 'Prototype chain'.
+   2. To go a bit deeper into both the chain and inheritance, spend some time with [this great article](http://javascript.info/prototype-inheritance). As usual, doing the exercises at the end will help cement this knowledge in your mind. Don't skip them! Important note: This article makes heavy use of `__proto__` which is not generally recommended. The concepts here are what we're looking for at the moment. We will soon learn another method or two for setting the prototype.
+2. You might have noticed us using the `this` keyword in object constructors and prototype methods in the examples above.
+Dmitri Pavlutin's excellent [article on the `this` keyword](https://dmitripavlutin.com/gentle-explanation-of-this-in-javascript/) is very comprehensive and covers how it changes in various situations. You should have a solid understanding of `this` after reading it. Pay special attention to the pitfalls mentioned in each section.
```

Comment:
"This article", "this great article", and "article on the `this` keyword", are not ideal link texts for a11y reasons.

Less than ideal link text is present throughout our curriculum, but should be addressed soon. Although, let's improve the link texts here while we're updating this lesson.

For example, the second link text could be improved to:
```
... spend some time with [JavaScript.Info's article on Prototypal Inheritance](link_here). ...
```

Reply:
> I understand that "This article" and "this great article" aren't ideal link texts and will improve them but I'd like to know what's wrong with "article on the this keyword" and how it could be improved in your opinion.

Yeah, "article on the this keyword" isn't bad for a link text. Although, it doesn't make a lot of difference but we could do something like "article on `this` keyword by Dimitri Pavlutin". 

If you're curious, you can read more about link text on [WebAIM's article on links and hypertext](https://webaim.org/techniques/hypertext/).

> Also, should I improve the link texts for the articles/videos in the "Additional Resources" section as well?

If you want to, but it's not necessary for this pull request.